src/test/java/github/grace000/erversay/response/ResponseTest.java

```diff
@@ -0,0 +1,29 @@
+package github.grace000.erversay.response;
+
+import org.junit.Test;
+import static junit.framework.TestCase.assertEquals;
+
+public class ResponseTest {
+
+    @Test
+    public void itHasADefaultCodeOf200() {
+        Response response = new Response().withCode(200).withStatus("OK");
+        String formattedResponse = response.build();
+        assertEquals(formattedResponse, "HTTP/1.1 200 OK\r\nContent-Length: 0\r\n\r\n");
+    }
```

Comment:
What would happen if you wrote another test that didn't have `200` being passed into `withCode`?

Reply:
The withCode method returns which ever code it receives as an argument. The default code is 200.

Reply:
Yeah, so now you have two tests that are testing against a 200 response. Are you 100% sure if you pass in a value that's *not* 200 that it'll return it?

Reply:
I deleted the extra test, but any string passed into withCode will be returned. The build method returns 200 by default if withCode is not used.